### PR TITLE
{bio}[foss/2018b] SqueezeMeta v0.4.3 w/ Python 2.7.15

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~#7712~~ (CheckM, pplacer), ~~#7729~~ (BLAST+), ~~#7738~~ (barrnap), ~~#7741~~ (DAS_tool), ~~#7742~~ (CD-HIT), ~~#7746~~ (MetaBAT), ~~#7763~~ (MinPath), ~~#7764~~ (PRINSEQ), ~~#7765~~ (AMOS), ~~#7767~~ (MaxBin)

### CODE REVIEW

**Comment by boegel** [comment]
Tested with the `SqueezeMeta.pl -m merged -p Hadza -s test.samples -f raw/` example command described at https://github.com/jtamames/SqueezeMeta/blob/master/INSTALL-CENTOS7 (which takes about 1.5 hours on 9 cores on a Skylake system), after pulling in the required databases using `$EBROOTSQUEEZEMETA/scripts/preparing_databases/download_databases.pl <path>`, seems to work as expected... :shipit:

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/54c84f07f7f5be8f5d3f60cf62e0bcff for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2532.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b7cf6ee4a1665a240db0e41e1520f25f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2093.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9a728fd4db96872ef1f7d987666e0c3a for a full test report.

File: easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-0.4.3-foss-2018b-Python-2.7.15.eb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    ('pplacer', '1.1.alpha19', '', True),
+]
+
+keepsymlinks = True
+
+postinstallcmds = [
+    # remove vendored tools that are include as dependencies
+    "rm -r %(installdir)s/bin/*",
+    # fix shebang in Perl scripts
+    "ls %(installdir)s/scripts/*.pl | xargs sed -i 's@#!/usr/bin/perl@#!/usr/bin/env perl@g'",
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-0.4.3-foss-2018b-Python-2.7.15.eb]: Not that it is important but why do you do it differently here? Why not simplye `sed -i .... %(installdir)s/scripts/*.pl` ?

Comment (boegel) [reply easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-0.4.3-foss-2018b-Python-2.7.15.eb]: I guess I didn't realize that you can pass `sed` multiple files at once, will fix.

Comment (boegel) [reply easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-0.4.3-foss-2018b-Python-2.7.15.eb]: fixed in 18fe362

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3103.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/8ec6f5fa681018a1389de35f762947d7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2068.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/195cd604d705dcc3f69baa6b939cdf74 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2400.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f00c04f942b275c3fb2a7aa5ae054542 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 22 out of 27 (1 easyconfigs in this PR)
nic424 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/f5ec17466dc1024fbac5bd89853e1968 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 22 out of 27 (1 easyconfigs in this PR)
nic411 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/12ec2daba991e08b8ea360b06ef92281 for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 14 out of 14 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/e7f65bc6c005d5328f6e02bb5774d592 for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @boegel!

**Pull request merged**